test/runnable/implicit.d

```diff
@@ -211,9 +211,9 @@ void test14155_for_testDIP29_4()
 {
     static assert( __traits(compiles, {              int*  p = make14155m (null); }));  //  m <- m (normal)
     static assert( __traits(compiles, {        const(int*) p = make14155m (null); }));  //  c <- m (normal)
-    static assert( __traits(compiles, {       shared(int*) p = make14155m (null); }));  // sm <- m (unique)
-    static assert( __traits(compiles, { shared(const int*) p = make14155m (null); }));  // sc <- m (unique)
-    static assert( __traits(compiles, {    immutable(int*) p = make14155m (null); }));  //  i <- m (unique)
+    static assert( __traits(compiles, {       shared(int*) p = make14155m (null); }));  // sm <- m (distinct)
```

Comment:
I actually think I got overzealous on the find/replace here... this is supposed to be unique isn't it because it is checking the implicit conversion of a unique value...

gonna undo this.